galaxy/importer/utils/spdx_licenses.json

```diff
@@ -0,0 +1,1148 @@
+{
```

Comment:
@awcrosby where did you take this json from?

Reply:
it comes from https://github.com/spdx/license-list-data/, using mazer to grab: https://github.com/ansible/mazer/blob/f55f14a64ea738082914a69ca47b9558915ea6ae/Makefile#L67

Reply:
Ah, I see, thanks :)